packages/cli/src/helpers.ts

```diff
@@ -0,0 +1,86 @@
+import { Commands } from '@prisma/internals'
+import Arg from 'arg'
+import fetch, { Headers } from 'node-fetch'
+
+export const platformParameters = {
+  global: {
+    '--early-access-feature': Boolean,
+    '--token': String,
+    '-t': '--token',
+  },
+  workspace: {
+    '--early-access-feature': Boolean,
+    '--token': String,
```

Comment:
Do we need to repeat global parameters for each command?

Reply:
Given how `arg` works, yes. Otherwise it complains about unknown parameters.

The other option would be to have an ancestor strip it, which brings its own complexity.

Reply:
Let's put a TODO for now.